tests/collections/asr/test_asr_classification_model.py

```diff
@@ -61,6 +61,7 @@ def speech_classification_model():
             'encoder': DictConfig(encoder),
             'decoder': DictConfig(decoder),
             'labels': ListConfig(["dummy_cls_{}".format(i + 1) for i in range(30)]),
+            'is_regression_task': False,
```

Comment:
Revert this since in most cases, older configs (and related tests) must pass without modification - so that older models like MatchboxNet and MarbleNet restoration is supported.

Reply:
++